doc/source/report/classification.rst

```diff
@@ -79,24 +90,36 @@ those values after encountering that.
 Data Requirements Description
 -----------------------------
 
-We dispose of a labeled data set of 16 different strains of mice.
-Behavioral features are recorded for each mouse and each day. One
-example can be the time spent eating or drinking, and the amount
-ingested.
-
-The researchers record the daily activities of each mouse, for example
-the time it spends eating, drinking, sleeping, and wondering around its
-habitats. Therefore, every behavioral features should be averaged to the
-same time period (one mouse day) for each mouse. For example, the food
-consumed variable at each timestamp will be aggregated to the average
-food consumption.
+For this project, we use behavioral data recorded for each mouse and each
+two-hour time bin of the day. There are in total 170 mice across 16 different
+strains. Behavioral features include measurements of daily activities, such 
+as food and water consumption, distance moved, the amount of time a mouse is 
+in active state, and the amount of time a mouse spends eating, drinking and 
+moving while in active state.
+
+For classification and clustering subsections, we require different processed 
+data.
+
+For classification project, we have two different dataset -- mouseday dataset 
+or individual mouse dataset. For the mouseday dataset, we take each 
+combination of mouse and day as a unique observation, resulting in  1921 
+observations. For the individual mouse dataset, we take the average measures 
+of different days for each mouse to form 170 unique observations. In each 
+dataset, we use the measures for different time bins as different features. 
+Therefore, we have a maximum of 9 types of features * 11 time bins = 99 features. 
+Users can also use a subset of these 99 features to train models. The mouse 
+strain number will serve as the label for the dataset.
+
+For clustering project, we used the individual mouse dataset which has 170 
+observations. For clustering, we do not want too many features since these 
+features might have directions very close to each other and it would be 
+repetitive to use all of them. Therefore, instead taking each hour bin as 
+different features like in the classification dataset, we take each hour bin 
+as different observations, resulting in only 9 features for the clustering 
+data. Then we take average of days for each mouse. While preparing the input 
+data, users will also have the choice to standardize data and/or add the 
+standard deviation for each of the 9 features, resulting in possibly 18 final features.
```

Comment:
Here we could mention that why we are not using the raw data with 21131 data points to do the clustering.